app/templates/components/search-facet-registration-type.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="registration-type-selector">
+    <i class='type-selector-warning' style='color: red; display: none'>Only available with OSF Registries</i>
```

Comment:
Not sure if this is intended, but the behavior of the warning is now a tooltip:
<img width="246" alt="screen shot 2017-05-23 at 1 34 49 pm" src="https://cloud.githubusercontent.com/assets/6268982/26367525/df00da7e-3fbc-11e7-8291-2b6cea657308.png">

while this has the old red warning.